### PR TITLE
Add document about creating backups

### PR DESCRIPTION
**Pull request by bkabrda**

### CODE REVIEW

File: docs/backups.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# OSBS Backups
+
+## Creating Backups
+
+Creating OSBS backups is very simple. To backup all relevant data, one needs to:
+
+1. Pause OSBS by `osbs pause-builds`
+2. Wait for all running builds to finish
```
Comment (twaugh) [inline docs/backups.md]: `osbs pause-builds` already performs step 2. Do we just need to say "..., which will prevent new builds being scheduled and wait for running builds to finish" in step 1?

File: docs/backups.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+3. Backup data using following commands.
+  * `oc export -o=json BuildConfig > buildconfigs.json`
```
Comment (twaugh) [inline docs/backups.md]: Nice! Although I tend to think `-o json` or `--output=json` looks better than `-o=json`. Is it just me?

Comment (mmilata) [inline docs/backups.md]: :+1:

File: docs/backups.md (hunk 1 above)
Comment (bkabrda) [inline docs/backups.md]: You're right, I'll fix that.

File: docs/backups.md (hunk 2 above)
Comment (bkabrda) [inline docs/backups.md]: I'll probably use `--output=json` to make the commands explicit, I guess that's best for documentation.

**Comment by bkabrda** [comment]
Good catch, I just blindly rewrote these and didn't see that. I'll push --force this, so that we don't get bazilion unnecessary commits...

**Comment by twaugh** [comment]
LGTM

**Pull request merged**